README.md

```diff
@@ -491,6 +491,11 @@ components and the relationship between them.
 localstack web
 ```
 
+## Other UI Clients
+
+* [Commandeer desktop app](https://getcommandeer.com)
+* (DynamoDB Admin Web UI)(https://www.npmjs.com/package/dynamodb-admin)
```

Comment:
nit: Can we please fix the markup for the second link

Reply:
Good catch, updated.